### PR TITLE
Fix & Clean up Organization Membership Modifications (Fixes #301)

### LINKED ISSUE #301: Admin role for organisations doesn't persist
Browser & Version: `Unknown`

Operating System: `Unknown`

Error message (if applicable): `None`

Steps to reproduce:

I'm currently `Developer` in the `EngineHub` organisation. Ever since the `Admin` rank was added, both the owner of the organisation and numerous people with global access on Ore have tried setting my rank to Admin. This has not applied each time.

* Try setting me to Admin on the EngineHub organisation
* It won't apply
* I'll still be developer

### PR DESCRIPTION
**Pull request by phase**
There was some whitespace being passed into an equals check, causing
it to fail.

Members that have been invited will now display what role they will be,
and that role is editable.

An NPE with Org PGP Keys was fixed by checking to see if it exists
before using it.

The modification icons were spaced out and the text was extracted into `conf/messages`
It looks a lot better now.

Fixes #301

### CODE REVIEW

**Comment by phase** [comment]
There's one more thing I want to fix.

**Comment by phase** [comment]
When editing a member, the role that is selected was set to the top, `Admin`. It will now be set to whatever the rank of the member is.

Should be good to merge!

**Review by me4502 — APPROVED** [review]

File: app/views/users/memberList.scala.html [hunk 1]
```diff
@@ -60,7 +60,8 @@
                                 aria-label="@messages("general.close")">
                             <span aria-hidden="true">&times;</span>
                         </button>
-                        <h4 class="modal-title" id="label-user-delete">@messages("project.removeMember")</h4>
+                        <h4 class="modal-title" id="label-use
+                        r-delete">@messages("project.removeMember")</h4>
```
Comment (kashike) [inline app/views/users/memberList.scala.html]: make this one line again

**Review by kashike — CHANGES_REQUESTED** [review]

File: app/views/users/memberList.scala.html (hunk 1 above)
Comment (phase) [reply app/views/users/memberList.scala.html]: Whoops, don't know how that typo got in there.

Comment (phase) [reply app/views/users/memberList.scala.html]: Typo fixed.

**Review by kashike — APPROVED** [review]

**Pull request merged**